controller/WFCORE-4868_configuration-files-aliases.adoc

```diff
@@ -0,0 +1,98 @@
+---
+categories:
+ - controller
+---
+= Aliases for standard configuration files
```

Comment:
@michpetrov We need to add the  [Community] word here in the feature title, that will help when this proposal is published on wildfly.org